### ISSUE TITLE
ios inspector无法获取我的界面元素

### ISSUE THREAD

**Issue by lyt19871224** [NetEaseGame/AutomatorX#81]
webdriveragent xcode运行报错：
2016-10-26 18:21:57.806 XCTRunner[93514:3571444] Enqueue Failure: UI Testing Failure - Failure fetching attributes for element <XCAccessibilityElement: 0x6000002410e0> Device element: Error Domain=XCTestManagerErrorDomain Code=13 "Error copying attributes -25202" UserInfo={NSLocalizedDescription=Error copying attributes -25202} <unknown> 0 1

**Comment by codeskyblue** [comment]
这个看起来像WDA的问题，不如提到这个地方去吧 https://github.com/facebook/WebDriverAgent
